### ISSUE TITLE
Allow easy download on CI servers

### ISSUE THREAD

**Issue by dotnetCarpenter** [MicrosoftEdge/edge-launcher#22]
This project is very useful but very difficult to use on Continues Integration (CI) servers, since its [npm package is outdated](https://www.npmjs.com/package/edge-launcher) (#20) and it is not available on [NuGet](https://www.nuget.org/packages?q=edge-launcher).

My use-case is to test [SVG.js](https://github.com/svgdotjs/svg.js/) on [AppVeyor using Karma](https://ci.appveyor.com/project/dotnetCarpenter/svg-js).

**Comment by nickmccurdy** [comment]
Have you gotten the npm package to work outside of karma-edge-launcher? I'm wondering if we can isolate the issue to this package, otherwise it would be a bug in karma-edge-launcher.

**Comment by dotnetCarpenter** [comment]
@nickmccurdy I haven't tried the npm package, but spotted your issue #20. I will give the npm package a shot. If it works, do you mind forking this project and release an updated npm package with another name?

**Comment by dotnetCarpenter** [comment]
@nickmccurdy I finally found out that the real issue is that I was using appveyor which has Windows Server 2012 as the default VM which doesn't have Edge. I'm hoping that we will get access to Windows Server 2016 and the issue will go away.

I would still prefer to use a newer version than [1.0.0.0}(https://github.com/MicrosoftEdge/edge-launcher/releases/tag/1.0.0.0), like [1.2.0.0](https://github.com/MicrosoftEdge/edge-launcher/releases/tag/1.2.0.0). But unless *someone* fork this project and publish it to npm...

**Comment by nickmccurdy** [comment]
Yeah, I created #20 when I noticed the npm package was out of date. The reason I didn't fix it myself is that the npm package's source is not available, and if I'm understanding this correctly it means I wouldn't be able to make a package with the same metadata. I'll double check this though.

Worst case, it probably would be a good idea to temporarily fork this to get up to date npm packages until it's fixed upstream. I can also help publish the new version of this package on npm if that's okay with someone from Microsoft with package access. My npm username is [nicolasmccurdy](https://www.npmjs.com/~nicolasmccurdy).
